themes/startwords/assets/scss/_utilities.scss

```diff
@@ -3,7 +3,7 @@
 // gridded sections use a 10-column grid with 10px gaps
 .grid {
     display: grid;
-    grid-template-columns: repeat(10, 1fr);
+    grid-template-columns: [full-start] repeat(10, 1fr) [full-end];
```

Comment:
whoa, I've never seen this syntax. what is it?

Reply:
I found it when I was trying to find a better way to do full-width in a grid! It lets you name a region and apply it. 

I created a new `full-width` class in the article scss that uses it like this:
```css
.full-width {
    width: 100vw;
    margin-left: (-$gutter);
    grid-column: full;
}
```

It made me wonder if we could be leveraging grid better / more semantically ...